x-pack/plugins/security_solution/server/endpoint/routes/actions/list_handler.test.ts

```diff
@@ -97,16 +97,13 @@ describe(' Action List Handler', () => {
 
   describe('Internals', () => {
     it('should return `notFound` when actions index does not exist', async () => {
-      mockDoesLogsEndpointActionsIndexExist.mockResolvedValue(false);
       await actionListHandler({ pageSize: 10, page: 1 });
-      expect(mockResponse.notFound).toHaveBeenCalledWith({
-        body: 'index_not_found_exception',
-      });
-    });
-
-    it('should return `ok` when actions index exists', async () => {
-      await actionListHandler({ pageSize: 10, page: 1 });
-      expect(mockResponse.ok).toHaveBeenCalled();
+      expect(mockGetActionList).toHaveBeenCalledWith(
```

Comment:
should you revert this change until we modify the existing List API?

Reply:
ahhh.. yes. Forgot to revert this one. Will do.